Automattic/jetpack
# Backup: minor styling fixes

#### Changes proposed in this Pull Request:
* adjusted margins for the connection screen
* brought back the "Where are my backups stored" on the Backup's "no plan" page
* made the "Connection" card match the styling of the "Heartbeat" card

#### Jetpack product discussion
p1HpG7-cGL-p2#comment-48286

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
0. Rebuild the Backup plugin if testing locally (`jetpack build plugins/backup`).
1. Disconnect Jetpack and go to the "Backup" page (connection screen). Confirm that the margins on the left screen are correct.
2. Connect with no plan. Confirm that the "Where are my backups stored" card shows up.
3. Confirm that the styling of the "Connection" card matches the "Heartbeat" card.
4. Confirm that the "Heartbeat" card does not have the "Activity" link.
5. Purchase a backup plan, confirm that you now see the "Activity" link in the "Heartbeat" card.
